doc/build.rst

```diff
@@ -161,6 +161,24 @@ To build with MinGW, type:
 
   cp make/mingw64.mk config.mk; make -j4
 
+Opening a can of worms - building XGBoost library for python for Windows with MinGW-w64
+---------------------------------------------------------------------------------------
+
+Windows versions of python are built with Microsoft Visual Studio. This has multiple issues:
+
+1. VS is proprietary and commercial software. While Microsoft provides a free edition (Community Edition), installing it may be illegal in your organization.
```

Comment:
Change this to  "VS is proprietary and commercial software. While Microsoft provides a free edition (Community Edition), your organization may prohibit its use because of its licensing terms."

Reply:
The problem is not in the organization. An org may have no policy about software at all.

Reply:
Did you mean that installing VS on a company computer may be against the licensing terms?

Reply:
FYI, according to https://visualstudio.microsoft.com/license-terms/mlt553321/,

> If you are an enterprise, your employees and contractors may not use the software to develop or test your applications, except for: (i) open source; (ii) Visual Studio extensions; (iii) device drivers for the Windows operating system; and, (iv) education purposes as permitted above

This is quite stringent restriction.

Reply:
If the MS licensing terms are problem, we can state that "While Microsoft provides a free edition (Community Edition), their licensing terms restrict when, where, and how it can be installed and used."